packages/quint-ui/src/components/label.tsx

```diff
@@ -0,0 +1,34 @@
+import clsx from "clsx";
```

Comment:
we should use `cn` instead so that tailwind classes are properly handled when they conflict

```suggestion
import { cn } from "../lib/utils.js";
```